modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/incubus_draft.dm

```diff
@@ -135,26 +136,32 @@
 	return ..()
 
 /datum/reagent/drug/aphrodisiac/incubus_draft/overdose_effects(mob/living/carbon/human/exposed_mob)
+	// Check if overdosing on incubus draft and succubus milk simultaneously, to prevent chat spam
+	var double_dosing = FALSE
+	for(var/r in exposed_mob.reagents.reagent_list)
+		var/datum/reagent/reagent = r
+		if(reagent.name == "succubus milk" && reagent.overdosed)
+			double_dosing = TRUE
+
+	// Begin cock growth
 	if(exposed_mob.client?.prefs?.read_preference(/datum/preference/toggle/erp/penis_enlargement))
+		// Start making new genitals if prefs allow it and if there isn't already one there
 		if(!exposed_mob.getorganslot(ORGAN_SLOT_PENIS) && exposed_mob.client?.prefs?.read_preference(/datum/preference/toggle/erp/new_genitalia_growth))
 			var/list/data = species_to_penis[exposed_mob.dna.species.id]
 			if(!data)
 				data = species_to_penis[SPECIES_HUMAN]
-
```

Comment:
why are you removing whitespace that's good for readability

Reply:
What's readable to some is not as clear to others I suppose...this looked the neatest to me. I put all the code related to the race-related organ information in one spot and then the space comes where the actual creation of new organs takes place. 

This is the only place in the code where I actually removed whitespace that wasn't already there, it was pretty darn unreadable to begin with. The comments I put and whitespace I added below I think are vast improvements. But if you would like I can put it back in this one single spot.